consensus/consensus_validator_test.go

```diff
@@ -48,6 +48,10 @@ func (MockChainReader) GetBlock(hash common.Hash, number uint64) *types.Block {
 	return &types.Block{}
 }
 
+func (MockChainReader) GetBlockByHash(hash common.Hash) *types.Block {
```

Comment:
Comment for a public function, please. If it is empty now, please add a TODO in the comment.

Reply:
Will do.

Reply:
Fixed in commit a8048ca0f923562b48478ce532e2086aa5ddff9b.